api/lib/polkadot-utils.js

```diff
@@ -0,0 +1,95 @@
+function extractState(
```

Comment:
can you add a description what this does?

Reply:
it extracts state :stuck_out_tongue:

Reply:
ok, done